courseraresearchexports/models/ExportDb.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python
+
+# Copyright 2016 Coursera
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import csv
+
+from sqlalchemy import create_engine
+from sqlalchemy.engine import reflection
+
+from courseraresearchexports.models.ContainerInfo import ContainerInfo
+
+
+class ExportDb:
+    """
+    Interface to SQLAlchemy for interacting with the a database
```

Comment:
Let's hide the fact that we are using `SQLAlchemy`. `ExportDb` as a class should provide functionality, what we use to talk to the DB is implementation detail.